nilearn/connectome/group_sparse_cov.py

```diff
@@ -386,7 +386,7 @@ def _group_sparse_covariance(
 
                         if fder == 0:
                             msg = "derivative was zero."
-                            warnings.warn(msg, RuntimeWarning)
+                            warnings.warn(msg, RuntimeWarning, stacklevel=4)
```

Comment:
This is not covered ? Do you wnat to address this is another PR ?

Reply:
sure but... one would first need to figure out how to even hit that line and this is the most (or one of the most) complex function in the codebase and between the abstract single letter variable and the nested debug statement, I kinda ran out of energy to refactor it to understand how to even get to that warning

so if anyone feels like trying to cover it: by all means be my guest

Reply:
Let's say it's for another PR ;-)